lib/livebook/intellisense/docs.ex

```diff
@@ -89,6 +90,7 @@ defmodule Livebook.Intellisense.Docs do
               kind: kind,
               name: name,
               arity: arity,
+              implicit: arity != base_arity,
```

Comment:
@josevalim let me know if there is a better name for a function generated from default args :)

Reply:
Went with `from_default`, thanks!